code.ts

```diff
@@ -1,10 +1,49 @@
 const delay = ms => new Promise(res => setTimeout(res, ms));
 var activeTimer = 0;
 const secondsSet = [86400, 3600, 60, 1];
+const progressBar = figma.createRectangle();
+var pause = 0;
+var reset = 0;
```

Comment:
Is  there a reason for not using a boolean here?

Reply:
just changed it to boolean